packages/blade/src/utils/useBreakpoint/useBreakpoint.ts

```diff
@@ -66,15 +68,17 @@ const useBreakpoint = ({ breakpoints }: { breakpoints: Breakpoints }): Breakpoin
   const getMatchedBreakpoint = useCallback(
     (event?: MediaQueryListEvent): Breakpoint => {
       const matchedBreakpoint =
-        breakpointsTokenAndQueryCollection.find(([_, __, mediaQuery]) => {
-          if (event?.media === mediaQuery) {
+        breakpointsTokenAndQueryCollection.find((breakpointTokenAndQuery) => {
```

Comment:
```suggestion
        breakpointsTokenAndQueryCollection.find(({ mediaQuery } ) => {
```

Nitpick

Reply:
need to add a lot of typings here since I'm returning either empty array or the array of objects with this structure so not worth the efforts